docs/source/diagrams/index.rst

```diff
@@ -0,0 +1,49 @@
+Plenum Diagrams
```

Comment:
Are the diagrams shown in the order specified here?
If so, I would suggest the following order:
1)  Consensus Protocol
2) Write Requests
3) Read Requests
4) Storages
5) Catchup Procedure
6) Catchup trigger
7) Checkpoints